### DISCUSSION TITLE
有几个问题请教

### DISCUSSION THREAD

**Discussion by MokOopsing** [rime/home#908] [Q&A] [answered]
1. 在使用```fluid_editor```时，能否设置为未输入时，数字及符号直接上屏，即数字和符号不作首位的效果？如果现在不能，后续能否参考这个想法作出修改？
2. ```contextual_suggestions: true ```是否对```enable_user_dict: true```有依赖？
3. 引入```grammar```，并设置```contextual_suggestions: true ``` ，但却不设置```max_homophones```或```max_homographs```，此时```grammar```是否生效？
4. rime中```delimiter```是否等同于分词键?

@lotem 希望能得到你的回答

**Comment by lotem** [comment, marked as answer]
​1. 可以設置爲無條件直接上屏。
語句流的設計類似電腦日文輸入，缺省地，數字及符號也會做轉換，當然也包括在句首時。
2. 沒有
3. 此參數未定義時默認值生效
4. 不分詞；切分音節、單個編碼

**Reply by MokOopsing** [reply]
多谢回复
